pubspec.yaml

```diff
@@ -15,4 +15,5 @@ dependencies:
   source_maps: ^0.10.2
 
 dev_dependencies:
+  source_span: ^1.4.1
```

Comment:
Make this constraint as wide as possible while still including all the features of `source_span` we use.

Reply:
Or just the latest supported by the current platform – since it has zero affect on version solving for any consumers – and doing otherwise seems like a profound amount of busy work...

Reply:
Following the prevailing style is important. Arguing about this is more work than looking up what APIs are used.